app/src/interfaces/useObservationApi.interface.ts

```diff
@@ -9,3 +11,25 @@ export interface IGetBlocksListResponse {
 export interface IGetObservationsListResponse {
   blocks: IGetBlocksListResponse[];
 }
+
+/**
+ * Create observation post object.
+ *
+ * @export
+ * @interface ICreateObservationRequest
+ */
+export interface ICreateObservationRequest {
+  metaData: IBlockObservationForm;
+  tableData: any[][];
```

Comment:
Heads up, there is a slight discrepancy between this definition, and the one in [Anissa's PR](https://github.com/bcgov/biohubbc/pull/367/files#diff-7978d58e06195e444e4da2d9adeda22c5478bdd772e82273c53b46f5babb5822R21).

I think we want to go with the way Anissa has it currently, where tableData is an object, with a data property that is a double array.  That way we can accommodate additional table related properties if we need to.